New Group/HomeScreen/ListMovieViewController.swift

```diff
@@ -0,0 +1,123 @@
+import UIKit
+
+fileprivate struct DefinesHeight {
+    static let movieTableRowHeight: CGFloat = 138
+}
+class ListMovieViewController: UIViewController {
+    
+    
+    @IBOutlet weak var categoryListCollectionView: UICollectionView!
+    @IBOutlet weak var listMovieTableView: UITableView!
+    @IBOutlet weak var searchTextField: UITextField!
+    @IBOutlet weak var searchButton: UIButton!
+    @IBOutlet weak var loadMoreButton: UIButton!
+    
+    var coordinator: ListMovieCoordinator!
+    var viewModel: ListMovieViewModel!
+    var arraySort = SortType.allCases
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configView()
+        fetchMoviesData()
+    }
+}
+
+//MARK: Fetch data
+extension ListMovieViewController {
+    private func fetchMoviesData() {
+        viewModel.fetchData()
+    }
+}
+//MARK: Configure view
+extension ListMovieViewController {
+    private func configView() {
+        configCollectionView()
+        configTableView()
+        configViewModel()
+        configTextField()
+    }
+    private func configTextField() {
+        searchTextField.attributedPlaceholder = NSAttributedString(
+            string: "Search Here...",
+            attributes: [NSAttributedString.Key.foregroundColor: UIColor.white]
+        )
+    }
+    private func configTableView() {
+        listMovieTableView.dataSource = self
+        listMovieTableView.delegate = self
+        listMovieTableView.registerNib(cellName: ListMovieTableViewCell.className)
+    }
+    
+    private func configCollectionView() {
+        categoryListCollectionView.delegate = self
+        categoryListCollectionView.dataSource = self
+        categoryListCollectionView.registerNib(cellName: CategoriesCollectionViewCell.className)
+    }
+    
+    func configViewModel() {
+        viewModel.notify = {
+            DispatchQueue.main.async { [weak self] in
+                self?.listMovieTableView.reloadData()
+            }
```

Comment:
As I said before, the [weak self] should be put at the first block of code so that you won't have retain cycle in closure.
In this case, put the [weak self] after the viewModel.notify = {